src/app/charts/charts.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="mt-3 justify-content-center">
+    <div class="btn-group btn-group-sm btn-group-toggle" data-toggle="buttons">
+        <label class="btn" [ngClass]="{
+            'btn-danger active':label === 'TotalConfirmed' || label === 'NewConfirmed',
+            'btn-success':label === 'TotalRecovered' || label === 'NewRecovered',
+            'btn-dark':label === 'TotalDeaths' || label === 'NewDeaths'
+          }" *ngFor="let label of labels">
+            <input type="radio" name="options" (click)="onClick(label)">
```

Comment:
We should somehow hook the label with the input

Reply:
The input is inside the label, or you ment something else?

Reply:
Ok, leave it as is!